lib/storage/leveldb.js

```diff
@@ -136,16 +137,23 @@ module.exports = class LevelDb {
       console.error(`can't store store inactive`);
     } else {
       try {
+        let time;
```

Comment:
voluntarily capture both  options.timesamp  and message arrival timestamp and  save them in the bucket.